src/chrome/chromeConnection.ts

```diff
@@ -155,7 +155,7 @@ export class ChromeConnection {
                 if (Array.isArray(responseArray)) {
                     // Filter out extension targets and other things
                     // Non-chrome scenarios don't always specify a type, so filter to include ones without a type at all
-                    let pages = responseArray.filter(target => target && (!target.type || target.type === 'page'));
+                    let pages = responseArray;//.filter(target => target && (!target.type || target.type === 'page'));
```

Comment:
Ok. Hm... it looks exactly like the thing I want to filter out when attaching to Chrome. I'm trying to figure out how to unblock nw.js debugging without impacting the experience for users attaching to Chrome, which is the only "officially supported" scenario. Let me think about it for a bit.